u7-components/src/router/index.ts

```diff
@@ -0,0 +1,37 @@
+import Vue from 'vue'
+import VueRouter, { RouteConfig } from 'vue-router'
+import Home from '@/views/Home.vue'
+import VideoPlayer from '@/views/VideoPlayer.vue'
+
+Vue.use(VueRouter)
+
+const routes: Array<RouteConfig> = [
+  {
+    path: '/',
+    name: 'Home',
+    component: Home
+  },
+  {
+    path: '/videos/:videoId',
+    name: 'VideoPlayer',
+    component: VideoPlayer
+  },
+  {
+    path: '/books',
+    name: 'BookSearch',
+    component: () => import('@/views/BookSearch.vue')
+  },
+  {
+    path: '/books/:isbn13',
+    name: 'BookDetails',
+    component: () => import('@/views/BookDetails.vue')
+  }
```

Comment:
ดี. FYI - ไวยากรณ์นี้จะทำให้การโหลดโค้ดล่าช้าจนกว่าจะจำเป็น สิ่งนี้เรียกว่า "การแยกโค้ด" (หรือการโหลดแบบขี้เกียจ)
Good. FYI - This syntax will delay loading the code until it's needed. This is called "code splitting" (or lazy-loading).